CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLWorker.cc

```diff
@@ -504,6 +519,19 @@ void SiPixelLorentzAnglePCLWorker::dqmAnalyze(edm::Event const& iEvent,
 
           SiPixelLorentzAngleTreeBarrel_->Fill();
 
+          int hindex = 2 * (layer_ - 1);
+          if (layer_ == 1) {
+            if (newmodule[layer_ - 1][ladder_ - 1][module_ - 1]) {
+              hindex += 1;
+            }
+          } else {
+            if (module_ > 4) {
+              hindex += 1;
+            }
+          }
+          if (hindex < 0 || hindex > 8)
+            continue;
```

Comment:
I don't understand the purpose of this section, can you explain it?

Reply:
This is copied and pasted from Morris's code to handle the new modules. I don't quite understand it either and also asked @tsusa and Morris. Do you think it's better to remove it for now and deal with it later when they reply?